Assets/_Scripts/ScriptableObjectDictionary.cs

```diff
@@ -0,0 +1,75 @@
+
+
+using UnityEngine;
+using System.Collections.Generic;
+
+public class ScriptableObjectDictionary : MonoBehaviour {
+
+    public Dictionary<Job, List<ActionBase>> playerJobActions = new Dictionary<Job, List<ActionBase>>();
+
+    public Dictionary<Job, List<ActionBase>> enemyJobActions = new Dictionary<Job, List<ActionBase>>();
+
+    [SerializeField] private JobActionsList[] _playerJobActionsLists;
+    [SerializeField] private JobActionsList[] _enemyJobActionsLists;
+
+
+    [SerializeField]
+    private List<StageInfo> stageInfo;
+
+    public void Initialize() {
```

Comment:
I disagree - The reason why Initialize is preferred over start or awake here is because we can guarantee the order of initialization. If we do it in Awake, we can't guarantee GameManager will be initialized before this class, and if we do it in Start, something in Awake() might be expecting this class to be initialized.

Reply:
Is it possible to remove the MonoBehaviour from `ScriptbleObjectDictionary`? Then you can use a constructor instead.

Reply:
It needs to be MonoBehaviour so you can assign the lists effortlessly